orienteer-core/src/main/java/org/orienteer/core/component/widget/browse/ListAllODocumentsWidget.java

```diff
@@ -38,20 +33,22 @@ public ListAllODocumentsWidget(String id, IModel<OClass> model,
 			IModel<ODocument> widgetDocumentModel) {
 		super(id, model, widgetDocumentModel);
 		IModel<DisplayMode> modeModel = DisplayMode.VIEW.asModel();
-		
-		Form<ODocument> form = new Form<ODocument>("form");
-		OQueryDataProvider<ODocument> provider = new OQueryDataProvider<ODocument>("select from "+getModelObject().getName());
```

Comment:
Mentioned above: this functionality recommended to be resided in OQueryDataProvider